content/docs/introduction/overview.md

```diff
@@ -41,25 +41,23 @@ optional:
 * the main [Prometheus server](https://github.com/prometheus/prometheus) which scrapes and stores time series data
 * [client libraries](/docs/instrumenting/clientlibs/) for instrumenting application code
 * a [push gateway](https://github.com/prometheus/pushgateway) for supporting short-lived jobs
-* special-purpose [exporters](/docs/instrumenting/exporters/) (for HAProxy, StatsD, Graphite, etc.)
-* an [alertmanager](https://github.com/prometheus/alertmanager)
+* special-purpose [exporters](/docs/instrumenting/exporters/) for services like HAProxy, StatsD, Graphite, etc
```

Comment:
etc.

Reply:
Will update - BTW it's the only sentence in the list ending in a period now.